fhir-bulkimportexport-webapp/src/main/java/com/ibm/fhir/bulkimport/ChunkWriter.java

```diff
@@ -149,16 +148,19 @@ public void writeItems(List<java.lang.Object> arg0) throws Exception {
                             chunkData.getBufferStream4Import().write(Constants.NDJSON_LINESEPERATOR);
                         }
                     }
-                } catch (FHIRPersistenceException e) {
+                } catch (Exception e) {
                     logger.warning("Failed to import due to error: " + e.getMessage());
                     failedNum++;
                     if (Constants.IMPORT_IS_COLLECT_OPERATIONOUTCOMES) {
                         FHIRGenerator.generator(Format.JSON).generate(FHIRUtil.buildOperationOutcome(e, false), chunkData.getBufferStream4ImportError());
                         chunkData.getBufferStream4ImportError().write(Constants.NDJSON_LINESEPERATOR);
                     }
                 }
-
             }
+            // Release the DB connection.
+            // This doesn't really commit the transaction, because the transaction was started and will be committed
+            // by the JavaBatch framework.
+            txn.commit2();
```

Comment:
It should use only txn.commit 

if a behavior needs to change, then change in txn.commit

Reply:
have not found a way to change inside commit without adding new field to the transaction to tell the commit to only drop it's connection without commit the user transaction. but I can do so if need.

Reply:
changed to close the connection inside existing commit method